ui/runs/forms/importing.py

```diff
@@ -126,3 +126,34 @@ class PeptideImportForm(MethodForm):
     intensity_name = CustomChoiceField(
         choices=IntensityType, label="Intensity parameter (same as MS-Data)"
     )
+    map_to_uniprot = CustomBooleanField(
+        label="Map to Uniprot IDs using Biomart (online)", required=False
+    )
+
+    def fill_form(self, run: Run) -> None:
+        from protzilla.methods.importing import MaxQuantImport, MsFraggerImport, DiannImport
+
+        self.fields["intensity_name"].initial = (
+            run.steps.get_step_input([MaxQuantImport, MsFraggerImport, DiannImport], "intensity_name"))
+
+        self.fields["map_to_uniprot"].initial = (
+            run.steps.get_step_input([MaxQuantImport, MsFraggerImport, DiannImport], "map_to_uniprot"))
+
+
+class EvidenceImportForm(MethodForm):
+    file_path = CustomFileField(label="Evidence file")
+    intensity_name = CustomChoiceField(
+        choices=IntensityType, label="Intensity parameter"
+    )
+    map_to_uniprot = CustomBooleanField(
+        label="Map to Uniprot IDs using Biomart (online)", required=False
+    )
+
+    def fill_form(self, run: Run) -> None:
+        from protzilla.methods.importing import MaxQuantImport, MsFraggerImport, DiannImport
+
+        self.fields["intensity_name"].initial = (
```

Comment:
Do the initials make sense? Is it possible to change if you want to `map_to_uniprot` between protein and peptide import?